v2/protocol/http/utility_test.go

```diff
@@ -89,3 +90,67 @@ func TestNewEventFromHttpResponse(t *testing.T) {
 		})
 	}
 }
+
+func TestNewEventsFromHTTPRequest(t *testing.T) {
```

Comment:
Can you add a test where `NewEventsFromHTTPRequest` returns an error and maybe another one with more than one event?

Reply:
Thanks for this suggestion! Ended up leading me to the code to see how easy it would be to do a `NewHTTPRequestFromEvent`, so added that in as well.

Updates to tests also implemented.

Reply:
Can you please add test cases which actually assert an error?